lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -463,19 +463,17 @@ private function createPropertyValueGenerator(
                 return $property->getTypeName() === 'bigint'
                     ? new Sequencing\BigIntegerIdentityGenerator($sequenceName)
                     : new Sequencing\IdentityGenerator($sequenceName);
-
             case GeneratorType::SEQUENCE:
                 $definition = $property->getValueGenerator()->getDefinition();
+
                 return new Sequencing\SequenceGenerator(
                     $platform->quoteIdentifier($definition['sequenceName']),
                     $definition['allocationSize']
                 );
-                break;
-
             case GeneratorType::CUSTOM:
                 $class = $property->getValueGenerator()->getDefinition()['class'];
+
                 return new $class();
-                break;
```

Comment:
It might be something to add to our CS as well: when a `break` is preceded by a `return`, it can be marked as useless 👍

Reply:
I could not find it in `slevomat/coding-standard` so I created issue for it https://github.com/slevomat/coding-standard/issues/636

Reply:
My bad, we already forbid it: https://github.com/doctrine/coding-standard/blob/master/lib/Doctrine/ruleset.xml#L420